src/main/java/edu/lwtech/csd297/tap4tap/pojos/Brewery.java

```diff
@@ -1,7 +1,7 @@
 package edu.lwtech.csd297.tap4tap.pojos;
 
 public class Brewery {
-    private String breweryId;
+    private int breweryId;
```

Comment:
This should not be an int, because the data from open brewery db does not use int for the id either.

we will use int for the primary keys in the other tables, though.

Reply:
ok! I see! We'll keep it to String?

Reply:
Yes, unless we want to use the [java.util.UUID](https://devdocs.io/openjdk~17/java.base/java/util/uuid) type.

Reply:
I think we should try to use UUID if we can; it will be more accurate. Done.